core/payment-driver/erc20/src/driver/cli.rs

```diff
@@ -0,0 +1,81 @@
+/*
+    Driver helper for handling messages from CLI.
+
+    Please limit the logic in this file, use local mods to handle the calls.
+*/
+// Extrnal crates
+
+// Workspace uses
+use ya_payment_driver::{
+    bus,
+    db::models::Network,
+    model::{Fund, GenericError, Init},
+};
+use ya_utils_futures::timeout::IntoTimeoutFuture;
+
+// Local uses
+use crate::{
+    dao::Erc20Dao,
+    driver::Erc20Driver,
+    erc20::{utils, wallet},
+    network, DRIVER_NAME,
+};
+
+pub async fn init(driver: &Erc20Driver, msg: Init) -> Result<(), GenericError> {
+    log::debug!("init: {:?}", msg);
+
+    // TODO: payment_api fails to start due to provider account not unlocked
```

Comment:
This could be easily resolved by checking mode in the Init msg. No need to comment out the whole thing.

Reply:
Bump.

Reply:
since this was copied from zksync i kept it for last, will fix it in both drivers after the refinement

Reply:
fixed